internal/logger/logger.go

```diff
@@ -341,7 +341,7 @@ func errToEntry(ctx context.Context, err error, errKind ...interface{}) log.Entr
 // consoleLogIf prints a detailed error message during
 // the execution of the server.
 func consoleLogIf(ctx context.Context, err error, errKind ...interface{}) {
-	if Disable {
+	if MinimumLogLevel > ErrorLvl {
```

Comment:
> Shouldn't we be doing a modulo operation here ?
No, if you set MinimumLogLevel to FatalLvl, then we need to avoid error logs.

So in testing mode, we set MinimumLogLevel to ErrorLvl to print only ErrorLvl and FatalLvl and not the unneeded info message. The only purpose of this PR is to avoid info message in our CI/CD failure reports. (if you remember seeing a lot of WARNING)